contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -9,6 +9,17 @@ Feature gates are intended to cover the development life cycle of a feature - th
 Features generally progress through `Alpha` -> `Beta` -> `GA`. Sometimes we end up deciding that a feature is not going to be supported and we end up marking them as `Deprecated`.
 
 The majority of features will go through all three stages, but occasionally there are features which may skip stages.
+While some exceptions may happen, approvers should use the following guidance:
+- features that involve API changes must progress through all `Alpha`, `Beta`, `GA` stages
+- larger or more complex changes should progress through all `Alpha`, `Beta`, `GA` stages
```

Comment:
```suggestion
- features with significant complexity, risk, or changes to performance or scalability characteristics of a cluster should progress through all `Alpha`, `Beta`, `GA` stages
```

(trying to expand the word "larger" into criteria somehow.. further improvements welcome)

Reply:
+1 to the clarification.

Reply:
I like this too - thanks!